.pre-commit-config.yaml

```diff
@@ -142,7 +142,7 @@ repos:
   - id: disallow-caps
     name: Disallow improper capitalization
     language: pygrep
-    entry: PyBind|Numpy|Cmake|CCache|PyTest
+    entry: PyBind|Cmake|CCache|PyTest
```

Comment:
From playing around for a couple minutes, I believe this will work, too:

```diff
-    entry: PyBind|Numpy|Cmake|CCache|PyTest
+    entry: PyBind|\bNumpy\b|Cmake|CCache|PyTest
```

Reply:
I’d prefer this, personally.

Reply:
Sure! Added it.